tests.js

```diff
@@ -100,15 +100,16 @@ describe('Order Book', () => {
       expect(updatedBook).to.deep.equal([{ type: 'sell', quantity: 12, price: 6950 }])
     })
 
-    it.skip('Extra Credit: it does not fulfill a mismatched order when it does not benefit both parties', () => {
+    it('Extra Credit: it does not fulfill a mismatched order when it does not benefit both parties', () => {
       const existingBook = [{ type: 'buy', quantity: 15, price: 5900 }, { type: 'sell', quantity: 12, price: 6950 },]
       const incomingOrder = { type: 'sell', quantity: 15, price: 6000 }
 
       const updatedBook = reconcileOrder(existingBook, incomingOrder)
 
       expect(updatedBook).to.deep.equal([
-        { type: 'buy', quantity: 15, price: 5900 },
+
         { type: 'sell', quantity: 12, price: 6950 },
+        { type: 'buy', quantity: 15, price: 5900 },
```

Comment:
Changing this order shouldn't be necessary. I told some folks it was ok to do but really you shouldn't have to.